packages/profile-sidebar/reducer.js

```diff
@@ -172,12 +170,12 @@ export default (state = initialState, action) => {
     }
     case `ORGANIZATION_SELECTED`: {
       const selectedOrganization = action.selected;
-      let { profiles } = state;
+      const { profiles } = state;
+      let filteredProfiles;
```

Comment:
should this be, I'm just consious that you are setting profiles: filteredProfiles futher down and the default value is an array
```suggestion
      let filteredProfiles = [];
```